src/Main.java

```diff
@@ -122,11 +128,21 @@ public static void main(String[] args) {
     }
 
     /***
-     * This fuction will take a string parameter and return 2d array of binary matrix.
+     * This functio will read a positive integer and determined how many dots exist in a pentagonal shape
+     * around a center dot on the Nth iteration.
+     * @param n The integer that is nth iteration
+     * @return integer that is Centered pentagonal number : Cpn = (5n^2-5n+2)/2
+     */
+    public static int pentagonalNumbers(int n){
+
+        return (int)((5*Math.pow(n,2)) - 5*n +2) / 2;
+    }
+    /***
+     * This function will take a string parameter and return 2d array of binary matrix.
      * @param str The string that will change
      * @return newArr The 2d array of the str
      */
-    private static int[][]  matrix (String str){
+    private static int[][] matrix (String str){
```

Comment:
reformat file to add appropriate spaces ){

Reply:
done. thank you